### ISSUE TITLE
Potential regression in yuzu with Minecraft: Story Mode

### ISSUE THREAD

**Issue by bunnei** [MerryMage/dynarmic#320]
Commit 2f567fe seems to introduce a regression in Minecraft: Story Mode. Prior to this commit, the game would boot to the title screen. After this commit, the game seems to softlock at the first loading sequence:

![image](https://user-images.githubusercontent.com/6956688/42856387-151fd136-8a13-11e8-8cc5-0d3aee87721e.png)

**Comment by MerryMage** [comment]
The following patch avoids the softlock:

```
 src/frontend/A64/translate/impl/floating_point_conversion_integer.cpp
@@ -146,13 +146,13 @@ bool TranslatorVisitor::FCVTZS_float_int(bool sf, Imm<2> type, Vec Vn, Reg Rd) {
     IR::U32U64 intval;
 
     if (intsize == 32 && *fltsize == 32) {
         intval = ir.FPSingleToFixedS32(fltval, 0, FP::RoundingMode::TowardsZero);
     } else if (intsize == 32 && *fltsize == 64) {
         intval = ir.FPDoubleToFixedS32(fltval, 0, FP::RoundingMode::TowardsZero);
     } else if (intsize == 64 && *fltsize == 32) {
-        intval = ir.FPSingleToFixedS64(fltval, 0, FP::RoundingMode::TowardsZero);
+        return InterpretThisInstruction();
     } else if (intsize == 64 && *fltsize == 64) {
-        intval = ir.FPDoubleToFixedS64(fltval, 0, FP::RoundingMode::TowardsZero);
+        return InterpretThisInstruction();
     } else {
         UNREACHABLE();
     }
@@ -173,13 +173,13 @@ bool TranslatorVisitor::FCVTZU_float_int(bool sf, Imm<2> type, Vec Vn, Reg Rd) {
     IR::U32U64 intval;
 
     if (intsize == 32 && *fltsize == 32) {
         intval = ir.FPSingleToFixedU32(fltval, 0, FP::RoundingMode::TowardsZero);
     } else if (intsize == 32 && *fltsize == 64) {
         intval = ir.FPDoubleToFixedU32(fltval, 0, FP::RoundingMode::TowardsZero);
     } else if (intsize == 64 && *fltsize == 32) {
-        intval = ir.FPSingleToFixedU64(fltval, 0, FP::RoundingMode::TowardsZero);
+        return InterpretThisInstruction();
     } else if (intsize == 64 && *fltsize == 64) {
-        intval = ir.FPDoubleToFixedU64(fltval, 0, FP::RoundingMode::TowardsZero);
+        return InterpretThisInstruction();
     } else {
         UNREACHABLE();
     }
```

TODO:

* Reduce the above patch to minimal required.
* Dump instruction sequences around the affected instructions.

**Comment by MerryMage** [comment]
294a1283adfb64676e619f31043780f1e99b0dc4 is a proposed fix.

**Comment by MerryMage** [comment]
Likely fixed by 5a91c94d (pushed to master).
